meilisearch-http/src/option.rs

```diff
@@ -248,11 +249,11 @@ impl Opt {
         // Parse the args to get the config_file_path.
         let mut opts = Opt::parse();
         let mut config_read_from = None;
-        if let Some(config_file_path) = opts
-            .config_file_path
-            .clone()
-            .or_else(|| Some(PathBuf::from("./config.toml")))
-        {
+        if let Some(config_file_path) = opts.config_file_path.clone().or_else(|| {
+            Some(PathBuf::from(
+                env::var("MEILI_CONFIG_FILE_PATH").unwrap_or_else(|_| "./config.toml".to_string()),
+            ))
+        }) {
```

Comment:
```suggestion
        let config_file_path = opts.config_file_path.clone()
        .or_else(|| env::var("MEILI_CONFIG_FILE_PATH").map(PathBuf::from).ok())
        .unwrap_or_else(|| PathBuf::from("./config.toml"));
        if let Some(config_file_path) = config_file_path {
```

I would prefer that you write that in multiple lines and then match on it after.

Reply:
Thanks for the suggestion. This is a log cleaner! Also I noticed we may not need the `if let Some` because there is always a fallback default path.